heft-plugins/pre-compile-hardlink-or-copy-plugin/package.json

```diff
@@ -12,15 +12,15 @@
     "build": "heft test --clean"
   },
   "peerDependencies": {
-    "@rushstack/heft": "^0.4.4"
+    "@rushstack/heft": ">=0.4.4 <1.0.0"
```

Comment:
This is isomorphic to what was there before.

Reply:
@iclanton That is not true. The `^` and `~` operators work weirdly for versions starting with `0.`

Reply:
Huh. TIL.